File: Kapsch.ITS.Gateway.Models/Ticket/CourtsInfoModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Kapsch.ITS.Gateway.Models.Ticket
{
    public class CourtsInfoModel
    {
        public IList<CourtModel> Courts { get; set; }
        public IList<CourtRoomModel> CourtRooms { get; set; }
        public IList<CourtDateModel> CourtDates { get; set; }
    }
}

File: Kapsch.MobileApps/KapschMobile/1_iTicket/src/main/java/za/co/kapsch/iticket/Models/CourtsInfoModel.java
package za.co.kapsch.iticket.Models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by CSenekal on 2017/01/25.
 */
public class CourtsInfoModel {

    @SerializedName("Courts")
    public List<CourtModel> mCourts;

    @SerializedName("CourtRooms")
    public List<CourtRoomModel> mCourtRooms;

    @SerializedName("CourtDates")
    public List<CourtDateModel> mCourtDates;

    public List<CourtModel> getCourts() {
        return mCourts;
    }

    public List<CourtRoomModel> getCourtRooms() {
        return mCourtRooms;
    }

    public List<CourtDateModel> getCourtDates() {
        return mCourtDates;
    }
}
